stonesoup/types/mixture.py

```diff
@@ -56,6 +61,40 @@ def append(self, component):
     def extend(self, new_components):
         return self.components.extend(new_components)
 
+    @property
+    def ndim(self):
+        if len(self.components):
+            return self.components[0].ndim
+        return 0
+
+    @property
+    def means(self):
+        return StateVectors([component.mean for component in self.components])
+
+    @property
+    def covars(self):
+        return np.stack([component.covar for component in self.components], axis=2)
+
+    @property
+    def weights(self):
+        return np.asarray([component.weight for component in self.components])
+
+    @property
+    def state_vector(self):
+        return self.mean
+
+    @property
+    def mean(self):
+        means = self.means
+        weights = self.weights / Probability.sum(self.weights)
+        return np.average(means, axis=1, weights=weights)
+
+    @property
+    def covar(self):
+        _, covar = gm_reduce_single(self.means, self.covars,
+                                    self.weights)
+        return covar
+
```

Comment:
I think it might be good to add a timestamp property, so that we can easily access the timestamp of the components.  This is quite important to avoid errors when calculating metrics, for example.  Something like:
`@property
def timestamp(self):
    return next((component.timestamp for component in self.components), None)`
Maybe a check to confirm all the component timestamps are equal would be sensible to add somewhere.